airflow/jobs.py

```diff
@@ -874,177 +882,222 @@ def _execute(self):
         not_ready = set()
         deadlocked = set()
 
-        for task in self.dag.tasks:
-            if (not self.include_adhoc) and task.adhoc:
-                continue
+        # create dag runs
+        dr_start_date = start_date or min([t.start_date for t in self.dag.tasks])
+        next_run_date = self.dag.normalize_schedule(dr_start_date)
+        end_date = end_date or datetime.now()
 
-            start_date = start_date or task.start_date
-            end_date = end_date or task.end_date or datetime.now()
-            for dttm in self.dag.date_range(start_date, end_date=end_date):
-                ti = models.TaskInstance(task, dttm)
-                tasks_to_run[ti.key] = ti
-                session.merge(ti)
-        session.commit()
+        active_dag_runs = []
```

Comment:
ah that makes sense. Im not sure if I know at the time of running whether we are a subdagoperator or not, but lets find out. Any suggestions how to report this back to the user? Just skip it?

On a side note @aoen , I probably can make backfill use the scheduler quite easily. However the side effect will be that backfill dont necessarily run at their point of origin. Is that a concern for you guys?